cmd/stack_lint.go

```diff
@@ -34,8 +37,6 @@ This command can be run from the base directory of your stack or you can supply
 	RunE: func(cmd *cobra.Command, args []string) error {
 
```

Comment:
Please remove extra space.

Reply:
This is added to separate the paragraphs when running `appsody stack lint --help` - I thought it looked better that way but I don't mind removing it.

Reply:
If you delete the line 35. It will not have any effect